FSJ_django20_project/FSJ/templates/FSJ/base_with_banner.html

```diff
@@ -1 +1 @@
-{% extends "FSJ/base.html" %}
{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}

{% block banner %}
    <div class="banner">
        <div>
            {# Translators: Home link on the main banner #}
            <a href='../home'>{% trans "Home" %}</a>
        </div>
        <div>
            {# Translators: Profile link on the main banner #}
            <a href='../profile'>{% trans "Profile" %}</a>
        </div>
        <div>
            {# Translators: Logout link on the main banner #}
            <a href='../logout'>{% trans "Logout" %}</a>
        </div>
    </div>
{% endblock %}
\ No newline at end of file
+{% extends "FSJ/base.html" %}
{% load i18n %} {# NOTE YOU MUST LOAD i18n to every HTML page even though it's also loaded to base.html #}

{% block banner %}
    <div class="banner">
        <div>
            {# Translators: Home link on the main banner #}
            <a class="header" href='/home'>{% trans "Home" %}</a>
        </div>
        <div>
            {# Translators: Profile link on the main banner #}
            <a class="header" href='/profile'>{% trans "Profile" %}</a>
        </div>
        <div>
            {# Translators: Students link on the main banner #}
            <a class="header" href='/studentlist'>{% trans "Students" %}</a>
        </div>
        <div>
            {# Translators: Logout link on the main banner #}
            <a class="header" href='/logout'>{% trans "Logout" %}</a>
        </div>

    </div>
{% endblock %}
```

Comment:
I'm not sure if we really want the main banner to include a link to students that only the coordinator can access. Maybe we can have this on the coordinator's home page or if we do want it to appear in the banner perhaps it can be shown only when the coordinator is the logged in user?